tools/parameter-curves/README.rst

```diff
@@ -0,0 +1,38 @@
+=========
+Parameter Curves
+=========
+
+This folder contains the code used to choose secure parameters using the Lattice-Estimator_.
+
+.. _Lattice-Estimator: https://github.com/malb/lattice-estimator
+
+In particular, we use data obtained from calls to the lattice estimator to generate parameter curves of the form:
+
+``sigma(n) = a * n + b``
+
+which can then be used to choose a suitable error standard deviation `sigma` for a given LWE dimension n. 
+
+Usage
+---------
+
```

Comment:
Maybe it could be good also to talk about the other scripts those which generate the code. Maybe for the usage we could start to describe the make targets as it instrument all the scripts with default parameters and so on, then we could explain the scripts generate_data.py.

Reply:
good point, I'll link the README to the makefile